src/view/components/tooltipClone.jsx

```diff
@@ -2,6 +2,19 @@ import React, { useEffect, useRef, useState } from 'react';
 import ReactDOM from 'react-dom';
 import './tooltipClone.styl';
 
+const isLabelTextfieldOnTop = (el) =>
```

Comment:
The tooltips were not aligned with the input fields when the label is above the fields. The changes in this file are fixing that.